### DISCUSSION TITLE
What is the correct StartupType for the WSSD Agent Service (wssdagent)?

### DISCUSSION THREAD

**Discussion by Nikkelmann** [Azure/aksArc#369] [Q&A] [answered]
We noticed that the WSSD Agent Service (wssdagent) in our cluster has the StartupType set to Manual which seems a bit weird.

Is this correct or should it be set to Automatic?

**Comment by Elektronenvolt** [comment, marked as answer]
@Nikkelmann - I've checked on a single node and multi node setups - it's always automatic by default. And I think that's the idea about it - always on.

```
Get-Service wssdagent | Select-Object -Property Name, Status,StartType

Name       Status StartType
----       ------ ---------
wssdagent Running Automatic
```
